### PR TITLE
Issue 2024

### PR DESCRIPTION
**Pull request by bethshook**
moved padding rules to individual components that need them

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -142,79 +142,18 @@ const StepCardHeader = styled.div`
...
-        --step-card-padding-top: calc(${breakpoints.margins.mobile} * 2);
-        --step-card-padding-side: ${breakpoints.margins.mobile};
+    ${breakpoints.mobile`
+      padding: calc(${breakpoints.margins.mobile} * 2) ${breakpoints.margins.mobile};
     `}
-
-
-    ${props => props.unpadded ? '.step-card-body' : '&'} {
-        padding: var(--step-card-padding-top) var(--step-card-padding-side);
-    }
```
Comment (bethshook) [inline src/components/Card.tsx:217]: I propose abandoning this rule for now as it complicates managing padding for child components and I didn't see any StepCards in tutor with it set to false. But let me know if you think we should try to make it work here.

Comment (jivey) [reply src/components/Card.tsx:217]: Agree, I think it was only true in one niche render case for Tutor if even that

File: src/components/FreeResponseInput.tsx [hunk 2]
```diff
@@ -82,12 +89,6 @@ export const FreeResponseTextArea = styled.textarea<{isOverWordLimit: boolean} &
 `;
 FreeResponseTextArea.displayName = 'OSFreeResponseTextArea';
 
-const SubmitBtn = styled(Button)`
-  align-self: flex-end;
-  margin: 4rem;
-  float: right;
-`;
-
```
Comment (jivey) [inline src/components/FreeResponseInput.tsx:90]: Ah good catch, the controls already get justified it seems

**Review by jivey — APPROVED** [review]
Looks good 👍

**Pull request merged**